velox/vector/BaseVector.cpp

```diff
@@ -543,6 +549,22 @@ VectorPtr newConstant(
       sizeof(T) /*representedByteCount*/);
 }
 
+template <>
+VectorPtr newConstant<TypeKind::SHORT_DECIMAL>(
+    variant& value,
+    vector_size_t size,
+    velox::memory::MemoryPool* pool) {
+  VELOX_UNSUPPORTED();
```

Comment:
this probably will be `VELOX_NYI()` since, in a later PR, we will have to support variant to decimal?